modelad_test_config.yaml

```diff
@@ -0,0 +1,17 @@
+destination: &dest syn51498092
+staging_path: ./staging
+gx_folder: none
+gx_table: none
+datasets:
+  - biomarkers:
+      files:
+        - name: biomarkers
+          id: syn61250724.1
+          format: csv
+      final_format: json
+      provenance:
+        - syn61250724.1
+      destination: *dest
+      custom_transformations: 1
+      column_rename:
+        agedeath: ageDeath
```

Comment:
Repeating Brad's comment from somewhere, ageDeath needs to be camel case: `age_death`. You'll have to change your test input files so this field is camel case too.

Reply:
I'm confused about this. Where do I need to make changes? The input csv column name is `age_death` and the Jira ticket outlining the output has it as `ageDeath`.

Reply:
you can change the name post-processing using `agora_rename` instead of `column_rename`, see e.g. the`neuropath_corr` dataset's config

Reply:
...but actually, let's keep the field name as age_death in the json file. I think that means you can just omit both 'column_rename` and `agora_rename` from the config. I'll explain why next time we sync, but I'll update your ticket now.

Reply:
this means you'll have to update this key in the expected json for the tests as well, sorry I didn't catch this sooner

Reply:
I will hold off on doing a final review until this change is implemented